nightshade-webclients.yml

```diff
@@ -99,10 +99,10 @@
         (omero_web_config_set_for_host | default({})))
       }}
 
-    omero_web_release: "{{ omero_web_release_override | default('5.7.0') }}"
-    omero_figure_release: "{{ omero_figure_release_override | default('4.3.1') }}"
+    omero_web_release: "{{ omero_web_release_override | default('5.8.0') }}"
+    omero_figure_release: "{{ omero_figure_release_override | default('4.3.2') }}"
```

Comment:
Should this be mirrored in `ome-dundeeomero.yml`?

Reply:
Good point! Fixed.